### PR TITLE
327 group f jf transfers

### PR DESCRIPTION
**Pull request by Elaine-Krauss-TCG**

### CODE REVIEW

File: schema/JF_TRAN_PAC_MEMO.json [hunk 1]
```diff
@@ -67,15 +68,17 @@
...
       "minLength": 0,
-      "maxLength": 12,
-      "pattern": "^[ -~]{0,12}$",
-      "examples": ["PAC_JF_MEM"],
+      "maxLength": 32,
+      "pattern": "^[ -~]{0,32}$",
+      "examples": [
+        "JF_TRANSFER_PAC_MEMOS"
+      ],
       "fec_spec": {
```
Comment (mjtravers) [inline schema/JF_TRAN_PAC_MEMO.json:76]: Since this field can only have one value, the "const" schema directive can be used and assign the value to "JF_TRAN_PAC_MEMO". See here for an example: https://github.com/fecgov/fecfile-validate/blob/develop/schema/INDV_REC.json#L64-L78

Also, no "S" on JF_TRANSFER_PAC_MEMO on line 74

File: schema/JF_TRAN_PAC_MEMO.json [hunk 2]
```diff
@@ -283,7 +286,7 @@
         "TYPE": "NUM-8",
         "REQUIRED": "X (error)",
         "SAMPLE_DATA": 20120615,
-        "VALUE_REFERENCE": "YYYYMMDD",
+        "VALUE_REFERENCE": null,
```
Comment (mjtravers) [inline schema/JF_TRAN_PAC_MEMO.json:289]: Why was this changed from YYYYMMDD to null?

File: schema/JF_TRAN_PAC_MEMO.json [hunk 3]
```diff
@@ -348,7 +351,7 @@
       "type": ["string", "null"],
       "minLength": 0,
       "maxLength": 9,
-      "pattern": "^[ -~]{0,9}$",
+      "pattern": "^C[0123456789]{0,9}$",
```
Comment (mjtravers) [inline schema/JF_TRAN_PAC_MEMO.json:354]: This can be changed to ^C[0-9]{0,9}$ which is more succinct. The same pattern can be applied to the FILER_COMMITTEE_ID_NUMBER

**Review by mjtravers — CHANGES_REQUESTED** [review]
Some changes are requested. In addition to the inline comments, the ENTITY_TYPE property can be changed to a "const" since it can only have one value "COM". See here for an example: https://github.com/fecgov/fecfile-validate/blob/develop/schema/INDV_REC.json#L137-L151

File: schema/JF_TRAN_PAC_MEMO.json (hunk 1 above)
Comment (Elaine-Krauss-TCG) [reply schema/JF_TRAN_PAC_MEMO.json:72]: Feedback incorporated

File: schema/JF_TRAN_PAC_MEMO.json (hunk 3 above)
Comment (Elaine-Krauss-TCG) [reply schema/JF_TRAN_PAC_MEMO.json]: Feedback incorporated

File: schema/JF_TRAN_PAC_MEMO.json (hunk 2 above)
Comment (Elaine-Krauss-TCG) [reply schema/JF_TRAN_PAC_MEMO.json]: Not entirely sure, to be honest...

**Review by mjtravers — APPROVED** [review]
